### PR TITLE
Adds test for InlineObjectSchemaValidator

### PR DESCRIPTION
**Pull request by Xtansia**
### Description
Adds test for InlineObjectSchemaValidator

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```
Comment (dblock) [inline build.sh:1]: Looks like this is some personal stuff that snuck in.

**Review by dblock — CHANGES_REQUESTED** [review]
Linter complained, https://github.com/opensearch-project/opensearch-api-specification/actions/runs/9024077713/job/24797246589?pr=289.

File: build.sh (hunk 1 above)
Comment (Xtansia) [reply build.sh]: Good catch, cleaned it up, `git add -A` strikes again.

**Comment by Xtansia** [comment]
> Linter complained, https://github.com/opensearch-project/opensearch-api-specification/actions/runs/9024077713/job/24797246589?pr=289.

@dblock Fixed this, editor hadn't actually saved/flushed some of my other changes at commit time 🤦

**Review by dblock — APPROVED** [review]

**Pull request merged**